### PR TITLE
Added background details on IP ranges to the 1.9 migration doc

### LINKED ISSUE #535: IP Ranges & Managed/Unmanaged allocation
Update and integrate into the current docs.

### PR DESCRIPTION
**Pull request by degville**
fixes #535

### CODE REVIEW

File: en/installconfig-upgrade-to-2.md [hunk 1]
```diff
@@ -17,11 +17,50 @@ Ubuntu to 16.04 LTS. In so doing, the MAAS database and MAAS configuration will
...
-a test environment (that mirrors your production environment) be upgraded first, to
-pre-empt any issues.
+Prior to the upgrade all packages should be updated
+(`sudo apt-get update && sudo apt-get dist-upgrade`). It is recommended that
+backups be made and that a test environment (that mirrors your production
+environment) be upgraded first, to pre-empt any issues.
 
+## IP range changes
+
+The concepts and terminology used to define IP ranges within MAAS change
```
Comment (evilnick) [inline en/installconfig-upgrade-to-2.md]: have changed?

File: en/installconfig-upgrade-to-2.md [hunk 2]
```diff
@@ -17,11 +17,50 @@ Ubuntu to 16.04 LTS. In so doing, the MAAS database and MAAS configuration will
...
+
+MAAS 1.9 allows users to configure their managed networks to use either
+[static][1-9-static] or dynamic ranges. A static range is used by MAAS to allocate
+automatic (static) IP addresses while a dynamic range is used solely for
+DHCP.
+
+Static ranges are deprecated in MAAS 2.x. Instead, MAAS assumes full control
+of the address space for a network it manages with two exceptions:
+
+- An optionally defined [reserved range][ip-range] has been defined.
```
Comment (evilnick) [inline en/installconfig-upgrade-to-2.md]: When an optionally...

File: en/installconfig-upgrade-to-2.md [hunk 3]
```diff
@@ -17,11 +17,50 @@ Ubuntu to 16.04 LTS. In so doing, the MAAS database and MAAS configuration will
...
+- [Managed allocation][managed-subnet] in MAAS 2.3+ is disabled.
```
Comment (evilnick) [inline en/installconfig-upgrade-to-2.md]: When...

Comment (degville) [reply en/installconfig-upgrade-to-2.md]: Not sure what happened to my whens, thanks!

File: en/installconfig-upgrade-to-2.md (hunk 1 above)
Comment (degville) [reply en/installconfig-upgrade-to-2.md]: aargh

**Pull request merged**